src/components/Notifications/NoticeBoard.tsx

```diff
@@ -18,44 +19,44 @@ import Loading from "@/components/Common/Loading";
 import Page from "@/components/Common/Page";
 
 import routes from "@/Utils/request/api";
+import query from "@/Utils/request/query";
 import { formatName, relativeTime } from "@/Utils/utils";
 
-import { UserBareMinimum } from "../Users/models";
-import { useGetNotifications } from "./GetNotifications";
-import { NotificationData } from "./models";
+import { NoticeData } from "./models";
 
 export const NoticeBoard = () => {
   const { t } = useTranslation();
-  const { data, loading } = useGetNotifications(routes.getNotifications, {
-    query: { offset: 0, event: "MESSAGE", medium_sent: "SYSTEM" },
+  const { data, isLoading } = useQuery({
+    queryKey: [
+      routes.getNotifications.path,
+      { offset: 0, event: "MESSAGE", medium_sent: "SYSTEM" },
+    ],
```

Comment:
why complicate the query key?

```suggestion
    queryKey: ["notices"],
```